testing/assertable_json.go

```diff
@@ -0,0 +1,100 @@
+package testing
+
+import (
+	"encoding/json"
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	contractstesting "github.com/goravel/framework/contracts/testing"
+	"github.com/goravel/framework/support/maps"
+)
+
+type AssertableJsonImpl struct {
```

Comment:
It's different packages with `AssertableJson interface`, so the struct name can be `AssertableJson` directly.

```suggestion
type AssertableJson struct {
```